data.json

```
@@ -850,5 +850,11 @@
     "githubUsername": "kaiwalyakoparkar",
     "imageUrl": "https://user-images.githubusercontent.com/53310847/117104479-d3350180-ad99-11eb-8152-2afaff67fe04.png",
     "issueNumber": 475
-  }
+  },
+  {
+    "name": "Sulthan Mohaideen",
+    "githubUsername": "SulthanNK",
+    "imageUrl": "https://user-images.githubusercontent.com/32130871/117547009-e4586980-b04a-11eb-8daa-baad5ffb4949.png",
+    "issueNumber": 482
+}
```

```suggestion
  }
```
Lining up with the other braces

---

Added the request changes in the file.